### ISSUE TITLE
Revise recommendation for using "tap" event with custom buttons

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#1470]
### Problem description
The `"tap"` event is currently recommended for usage with custom buttons (see `imageview-as-a-button` snippet).

`"tap"` is currently only triggered when the finger briefly touches the widget without dragging. Slightly dragging while touching the widget results into  the `"tap"` event not being fired. [1] ([related issue](https://github.com/eclipsesource/tabris-js/issues/1467)) 

`"tap"` also would not fire if the finger touches the widget, drags outside it, then returns inside it and releases (default button behaviour on iOS). [2]

### Expected behavior
`"touchEnd"` may be a better alternative to `"tap"` for pressing interaction with custom buttons. It triggers for (2) and it does not have the issue from (1).

### Code snippet

```javascript
const {Composite, TextView, ui} = require('tabris');

let touched = 0;
new Composite({
  background: 'red',
  width: 200, height: 200,
  centerX: 0, centerY: 0,
  highlightOnTouch: true
}).on('touchEnd', () => {
  touched++;
  touchedLabel.text = 'touched ' + touched + ' times';
}).appendTo(ui.contentView);
let touchedLabel = new TextView({
  top: 'prev() 10', centerX: 0
}).appendTo(ui.contentView);
```

**Comment by ralfstx** [comment]
I agree that we should do something about it, but not sure that advertising the low-level touch API is the right solution. We discussed a dedicated "select" event that does also accepts small movements of the finger. We might also consider adjusting the behavior of the `tap` event. Let's discuss this in #1467.
